torch/csrc/jit/passes/alias_analysis.cpp

```diff
@@ -496,6 +513,66 @@ void AliasDb::analyzeChunk(Node* node) {
   }
 }
 
+void AliasDb::analyzeWait(Node* node) {
+  // Do nothing, this node should have already been analyzed by the
+  // corresponding analyzeFork(). See that function for details.
+}
+
+void AliasDb::analyzeFork(Node* node) {
+  // We treat forks as if their subgraph ran immediately. Since the alias
```

Comment:
Maybe rephrase ? We're treating forks as potentially running any time being fork & wait